client/my-sites/plugins/plugins-manage-mixin.js

```diff
@@ -7,14 +7,10 @@ module.exports = {
 		if ( ! selectedSite ) {
 			return true;
 		}
-		if ( ! selectedSite || ! selectedSite.modulesFetched ) {
-			return true;
-		}
-		if ( ! selectedSite.getModule( 'manage' ) ||
-				! selectedSite.getModule( 'manage' ).active ) {
+		if ( ! selectedSite.isModuleActive( 'manage' ) ) {
```

Comment:
Since the modules are now returned part of the `Site` object, I don't think the scenario you describe is possible.